ui/app/components/job-status/allocation-status-block.hbs

```diff
@@ -13,6 +13,9 @@
           @label="View allocation"
         >
           {{#if (and (eq @status "running") (not @steady))}}
+            {{#if (eq @canary "canary")}}
```

Comment:
`question`: Where are you injecting this argument?

Reply:
This is injected in the nested `each-in` blocks of allocation-status-row.hbs: https://github.com/hashicorp/nomad/blob/f64a0b51272e9a6bce9cff09bd2596a673a7cc0e/ui/app/components/job-status/allocation-status-row.hbs#L11-L18